src/main/java/mods/eln/misc/Version.java

```diff
@@ -1,74 +1,76 @@
 package mods.eln.misc;
 
+import mods.eln.Tags;
+import org.semver4j.Semver;
+
 import static mods.eln.i18n.I18N.tr;
 
 /**
  *
  * ====================================================================================
- *
  * WARNING! DO NOT CONVERT THIS TO KOTLIN WITHOUT CHECKING HOW GRADLE PARSES THIS FILE!
- *
  * ====================================================================================
- *
  * Current mod version. Used to check if a new mod version is available. Must be
  * set correctly for each mod release.
  *
  * @author metc
  */
 public final class Version {
 
+    /**
+     * SemVer Version
+     */
+    public final static Semver SEMVER = Semver.parse(Tags.VERSION);
+
     /**
      * Major version code.
      */
-    public final static int MAJOR = parseVersion("@MAJORVERSION@");
+    public final static int MAJOR = SEMVER.getMajor();
 
     /**
      * Minor version code.
      */
-    public final static int MINOR = parseVersion("@MINORVERSION@");
+    public final static int MINOR = SEMVER.getMinor();
 
     /**
      * Revision version code.
      */
-    public final static int REVISION = parseVersion("@REVISION@");
+    public final static int REVISION = SEMVER.getPatch();
 
-    public final static String BUILD_HOST = parseAdditionals("@BUILD_HOST@");
-    public final static String BUILD_DATE = parseAdditionals("@BUILD_DATE@");
-    public final static String JAVA_VERSION = parseAdditionals("@JAVA_VERSION@");
-    public final static String GIT_REVISION = parseAdditionals("@GIT_REVISION@");
+    // These attributes make the build not reproducible, should not bake in build-host information
+    //public final static String BUILD_HOST = parseAdditionals("@BUILD_HOST@");
+    //public final static String BUILD_DATE = parseAdditionals("@BUILD_DATE@");
+    //public final static String JAVA_VERSION = parseAdditionals("@JAVA_VERSION@");
+    public final static String BUILD_HOST = "";
+    public final static String BUILD_DATE = "";
+    public final static String JAVA_VERSION = "";
```

Comment:
We can just delete these.

Reply:
I figured, but some code references them, only in one place, but it would need to be deleted as well, and I didn't know if you wanted to do something else, but deleting it is trivial.